main/upower/APKBUILD

```diff
@@ -1,17 +1,23 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=upower
-pkgver=0.99.7
+pkgver=0.99.10
+_distver="$(echo ${pkgver} | tr '.' '_')"
 pkgrel=0
 pkgdesc="Power Management Services"
 url="https://upower.freedesktop.org"
 arch="all"
 license="GPL-2.0-or-later"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-makedepends="linux-headers gtk+-dev libgudev-dev libusb-dev polkit-dev
-	dbus-glib-dev libxslt gobject-introspection-dev docbook-xsl"
-source="https://upower.freedesktop.org/releases/upower-$pkgver.tar.xz"
-# XXX: tests are broken on alpine for some reason, but not on adelie
-options="!check"
+makedepends="linux-headers gtk+-dev libgudev-dev libusb-dev polkit-dev gtk-doc
+	dbus-glib-dev libxslt gobject-introspection-dev docbook-xsl autoconf automake
+	py3-dbus"
+source="https://gitlab.freedesktop.org/upower/upower/-/archive/UPOWER_${_distver}/upower-UPOWER_${_distver}.tar.bz2"
```

Comment:
```suggestion
source="https://gitlab.freedesktop.org/${pkgname}/${pkgname}/-/archive/${_pkgname}/${pkgname}-${_pkgname}.tar.bz2"
```

Reply:
I've been asked to write pkgname out here by @Ikke in another PR

Reply:
You mean, not to use the variable? Did you get an explanation as to why?

Reply:
Uh..I can search for it tomorrow, not quite sure what PR it was in.

Reply:
The upstream package name is separate from the aports package name, they can change for different reasons. Therefore it's not recommended to use `pkgname` in the source url and other places that is not controlled by the APKBUILD.

One usecase is when you would create a variant of this package (so with a different package name). Then suddenly the build breaks because the url is incorrect.

Reply:
So, let's solve the usecase you're concerned about without violating DRY.

Reply:
I tend to prefer try avoid use variables in URLs because it makes it easier to copy the URL (or part of the URL) paste to browser in addition to the reason @_ikke_ mentioned. I think the value of those outweights the value of DRY.